csharp/private/runtime.BUILD

```diff
@@ -13,3 +18,26 @@ exports_files(
     ]),
     visibility = ["//visibility:public"],
 )
+
+cc_binary(
+    name = "dotnetw",
+    srcs = [":main-cc"],
+    data = glob(
+        [
+            "dotnet",
+            "dotnet.exe",
+        ],
```

Comment:
Could you run this through buildifier?

Reply:
I think this will need `allow_empty = True` (to pass with `--strict`)

Reply:
It does have an issue with empty globs when running `bazel build --incompatible_disallow_empty_glob :*`. I'll add the `allow_empty` flag to these usages.

If anyone is interested in the list, it is [here](https://docs.bazel.build/versions/master/command-line-reference.html#flag--incompatible_disallow_empty_glob)